spec/jsSpecs/ares-device.spec.js

```diff
@@ -143,8 +143,6 @@ describe(aresCmd + ' --capture-screen(-c)', function() {
             if (stderr && stderr.length > 0) {
                 common.detectNodeMessage(stderr);
             }
-            expect(stdout).not.toContain(options.device);
```

Comment:
why do you remove this codes?

Reply:
I added "Set target device to oseTarget" for this commit.
"expect(stdout).not.toContain(options.device)" makes always failure.